README.md

```diff
@@ -86,107 +86,104 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
+
+#### Install using PyTorch without CUDA bindings and no GPU acceleration
+
+- Install on a standard Linux machine with:
+
+   ```shell
+   python<version> -m venv --upgrade-deps venv
+   source venv/bin/activate
+   pip install instructlab
+   ```
 
-   #### Install with Apple Metal on M1/M2/M3 Macs
+   *Additional Build Argument for Intel Macs*
+   If you have an Mac with an Intel CPU, you must add a prefix of
+   `CMAKE_ARGS="-DLLAMA_METAL=off"` to the `pip install` command to ensure
+   that the build is done without Apple M-series GPU support.
+   `(venv) $ CMAKE_ARGS="-DLLAMA_METAL=off" pip install ...`
 
-      > **NOTE**: Make sure your system Python build is `Mach-O 64-bit executable arm64` by using `file -b $(command -v python)`,
-      > or if your system is setup with [pyenv](https://github.com/pyenv/pyenv) by using the `file -b $(pyenv which python)` command.
-      >
-      > **NOTE**: Also, ensure that both `python3 -c 'import platform; print(platform.machine())'` and `arch` return `arm64`. If `arch` returns `i386` or `python3` returns `x86_64`, it means your terminal or Python is running in Intel emulation mode, which isn't optimized for Apple Silicon. This is especially important if you have synced data from an Intel-based Mac to an Apple Silicon Mac (M1/M2/M3), as you may need to review and update your environment, especially tools like `Homebrew` or `Bash` that were previously installed under Intel.
+#### Install with AMD ROCm
 
-      ```shell
-      python3 -m venv --upgrade-deps venv
-      source venv/bin/activate
-      pip cache remove llama_cpp_python
-      pip install 'instructlab[mps]'
+- Install on AMD ROCm with:
+
+   ```shell
+   python3 -m venv --upgrade-deps venv
+   source venv/bin/activate
+   pip cache remove llama_cpp_python
+   CMAKE_ARGS="-DLLAMA_HIPBLAS=on \
+      -DAMDGPU_TARGETS=all \
+      -DCMAKE_C_COMPILER=/opt/rocm/llvm/bin/clang \
+      -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++ \
+      -DCMAKE_PREFIX_PATH=/opt/rocm \
+      -DLLAMA_NATIVE=off" \
+      pip install 'instructlab[rocm]' \
+      --extra-index-url https://download.pytorch.org/whl/rocm6.0
       ```
 
-   #### Install with Nvidia CUDA
+#### Install with Nvidia CUDA
 
-     For the best CUDA experience, installing vLLM is necessary to serve Safetensors format models.
+- Install on Nvidia CUDA with:
 
-      ```shell
-      python3 -m venv --upgrade-deps venv
-      source venv/bin/activate
-      pip cache remove llama_cpp_python
-      CMAKE_ARGS="-DLLAMA_CUDA=on -DLLAMA_NATIVE=off" pip install 'instructlab[cuda]'
-      pip install vllm@git+https://github.com/opendatahub-io/vllm@2024.08.01
+   ```shell
+   python3 -m venv --upgrade-deps venv
+   source venv/bin/activate
+   pip cache remove llama_cpp_python
+   CMAKE_ARGS="-DLLAMA_CUDA=on -DLLAMA_NATIVE=off" pip install 'instructlab[cuda]'
+   pip install vllm@git+https://github.com/opendatahub-io/vllm@2024.08.01
```

Comment:
I think this might be out of date - e.g. we have a `requirements-vllm-cuda.txt` file now - @cdoern @n1hility can either of you confirm?

Reply:
yeah, but thats only if people clone the repo

Reply:
hmmm no way to pip install with VLLM? shouldn't it be part of the CUDA target?

Reply:
@nathan-weinberg is the PR your working on related to this?

Reply:
i'm only changing the tag, my points in this thread remain

Reply:
gotcha! @cdoern do you have insight on this?